src/tests/suite/dfuse_test.c

```diff
@@ -35,6 +35,7 @@ print_usage()
 {
 	printf("DFuse tests\n");
 	printf("dfuse_test --test-dir <path to test>\n");
+	printf("dfuse_test -m|--metadata\n");
```

Comment:
If this is needed then I would prefer that all tests were enabled by default and if necessary then an option was used to disable some of them.  It would probably also be nicer if we still ran the "tests" but only reported the results to stdout rather than failing.

Reply:
Then we should probably model it after `daos_test`, which runs all tests by default, but has flags to optionally run specific tests. I think the concept of disabling rather than enabling becomes confusing as more tests are added.

> still ran the "tests" but only reported the results to stdout rather than failing

I think that would require a bit of refactoring.

Anyway, this test wasn't quite set up like some of the other unit tests, which is why I took this simple approach.

Reply:
you probably have more experience of it than me, I'm just keen that we don't setup a situation where we can accidentally disable tests without noticing.  Happy to defer to you on this one.

Reply:
I overhauled the interface based on `dfs_test`. By default, all tests are ran, but each can be ran independently. `dfs_test` does have a `-E|--exclude` option, but it doesn't work, so I didn't include that.